src/apps/common/i18n/en-US.json

```diff
@@ -755,6 +755,16 @@
 			"textarea": {
 				"placeholder": "New Message..."
 			}
+		},
+		"tabs": {
+			"account": "Account",
+			"agent": "Agent",
+			"descendants": "Descendants"
+		},
+		"subtabs": {
```

Comment:
I meant that you should store all tab related strings under the same object, so you can move everything from `subtabs` under `tabs`.